.changeset/popular-ads-visit.md

```diff
@@ -0,0 +1,44 @@
+---
+"@hashicorp/flight-icons": patch
+---
+
+Add new Service icons to Flight
```

Comment:
I think these two are missing from the list:
- auth0
- auth0-color

Reply:
They aren't missing. I noticed they weren't respecting the alpha order and changed their order/position in the grid, but they are still in the set.

> {
      "id": "707:89",
      "fileName": "auth0-24",
      "iconName": "auth0",
      "description": "auth0",
      "size": "24",
      "width": 24,
      "height": 24
    },
    {
      "id": "707:92",
      "fileName": "auth0-16",
      "iconName": "auth0",
      "description": "auth0",
      "size": "16",
      "width": 16,
      "height": 16
    },
    {
      "id": "707:83",
      "fileName": "auth0-color-24",
      "iconName": "auth0-color",
      "description": "auth0",
      "size": "24",
      "width": 24,
      "height": 24
    },
    {
      "id": "707:86",
      "fileName": "auth0-color-16",
      "iconName": "auth0-color",
      "description": "auth0",
      "size": "16",
      "width": 16,
      "height": 16
    },

Reply:
👍